src/main/java/equipments/uniform/AbstractUniformEquipment.java

```diff
@@ -0,0 +1,30 @@
+package equipments.uniform;
+
+import equipments.IEquipment;
+
+public abstract class AbstractUniformEquipment implements IEquipment {
+    public int size;
+    public String uniformEquipmentType;
+
+    public AbstractUniformEquipment(int size){
+        this.size = size;
+    }
+
+    public int getSize() {
+        return size;
+    }
+
+    public void setSize(int size) {
+        this.size = size;
+    }
+
+    @Override
+    public String getEquipmentType() {
```

Comment:
Code duplication- this code exists both on `AbstractUniformEquipment` and on `AbstractTechnologyEquipment` (and presumably would have to be added to any other equipment type that may be added).